### PR TITLE
Servants of Ratvar get an alert if they have no tinkerer's cache

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Servants of Ratvar get an alert if they have no tinkerer's caches constructed. Because it's important that they should.
/:cl:

### CODE REVIEW

**Comment by ChangelingRain** [comment]
Tbh I think this alert should only fire if above 5 servants but doing that would be... tricky.

**Comment by coiax** [comment]
Caches are unlocked from tier 1, even a solo servant with a slab will eventually get two alloy.

**Comment by ChangelingRain** [comment]
also it's Tinkerer's Cache, not Clockwork Cache.
gotta be consistent.

File: code/_onclick/hud/alert.dm [hunk 1]
```diff
@@ -240,6 +240,14 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
 	icon_state = "blobbernaut_nofactory"
 	alerttooltipstyle = "blob"
 
+// CLOCKCULT
+/obj/screen/alert/nocache
+	name = "No Clockwork Cache"
+	desc = "In order to share components and unlock higher tier \
+		scripture, a clockwork cache must be constructed somewhere \
```
Comment (ChangelingRain) [inline code/_onclick/hud/alert.dm]: Here, and the line above, too

**Pull request merged**